pkg/tsdb/azuremonitor/applicationinsights-datasource_test.go

```diff
@@ -92,7 +91,7 @@ func TestApplicationInsightsDatasource(t *testing.T) {
 						"metricName":          "Percentage CPU",
 						"alias":               "testalias",
 						"queryType":           "Application Insights",
-						"allowedTimeGrainsMs": []interface{}{"auto", json.Number("60000"), json.Number("300000")},
+						"allowedTimeGrainsMs": []int64{60000, 300000},
```

Comment:
This could be a breaking change? But I looked at it with Ryan and it seems it is more of legacy thing. And I couldn't get queries where this property had an "auto" in it.